starlite/contrib/sqlalchemy/init/__init__.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional
+
+from starlite.di import Provide
+from starlite.exceptions import MissingDependencyException
+
+from .config import SQLAlchemyConfig
+
+try:
+    import sqlalchemy  # noqa: F401
+except ImportError as e:
```

Comment:
tricky one - as without the try except it will get isorted above the `.config` import.. prob means that the package needs to be structured better overall..

I'm going to move this stuff into main and it will be refactored based on the discussion in #1381 - so will tidy it up then.